packages/eslint-plugin/src/rules/no-misused-spread.ts

```diff
@@ -0,0 +1,47 @@
+import {
+  ESLintUtils,
+  type TSESLint,
+  type TSESTree,
+} from '@typescript-eslint/utils';
+
+import { createRule } from '../util';
+
+export default createRule({
+  defaultOptions: [],
+  name: 'no-misused-spread',
+  meta: {
+    docs: {
+      description: 'Disallow spread operator on function',
```

Comment:
should probably be updated to a more generic "no-spread" description

Reply:
I've proposed a change.